docs/Building_Diagrams/multiinstance.md

```diff
@@ -0,0 +1,110 @@
+
+## **Parallel Execution**
+
+All instances of the task are launched simultaneously, allowing for concurrent processing of the collection elements. In case of a parallel multi-instance activity, all instances are created when the multi-instance body is activated. The instances are executed concurrently and independently from each other.
+
+![Multi_instance_parallel](images/multiinstance_parallel_example.png)
+
+## Components of Multi-Instance Tasks
+Multi-instance tasks comprise several key properties that define their behavior:
+
+```{image} ./images/multiinstance_properties.png
+:width: 230px
+:align: right 
+``` 
+1. **Input Collection**: Specifies the array or collection over which the task iterates. Each element in the collection serves as input for a task instance.
+
+
+2. **Output Collection**: Collects the outcomes from all task instances into a single collection, enabling aggregation of results.
+
+3. **Loop Cardinality**: Defines the exact number of times the task should iterate. This is used when the number of instances is known ahead of time and is fixed.
+
+4. **Input Element Variable**: Represents each element in the input collection during an iteration, allowing for individual processing.
+
+5. **Output Element Variable**: Captures the result of each task instance, contributing to the output collection.
+
+6. **Completion Condition**: An optional condition that, when evaluated as true, can prematurely terminate the iterations.
+
+
+## Example: Using Multi-Instance Tasks for Dynamic Iteration
+
+This example outlines a BPMN process that demonstrates the use of a multi-instance task to iterate over and modify elements within a collection. 
+
+Specifically, the process manages a list of composers, their names, and genres, showcasing the dynamic handling of data objects through script and manual tasks.
+
+### Process Overview:
+
+#### 1. **Start Event**: 
+
+Marks the initiation of the process.
+
+![Multi_instance_example](images/multiinstance_example1.png)
+
+#### 2. **Script Task - Create Dictionary**:
+
+
+This task initializes a list (array) of dictionaries, each representing a composer with their name and associated genre. The script effectively sets up the data structure that will be manipulated in subsequent steps of the process.
+
+![Multi_instance_example](images/multiinstance_example2.png)
+
+**Script**:
+
+```python
+    composers = [
+       {"composer": "Johann Sebastian Bach", "genre": "Baroque"},
+       {"composer": "Ludwig van Beethoven", "genre": "Classical/Romantic"},
+       {"composer": "Wolfgang Amadeus Mozart", "genre": "Classical"}
+     ]
+```
+
+#### 3. **Multi-Instance Task - Edit Composer**:
+
+This task is configured as a parallel multi-instance task that iterates over the `composers` array created by the previous script task. It allows for the editing of each composers information within the array.
+
+![Multi_instance_example](images/multiinstance_example3.png)
```

Comment:
same here; i'd probably only use the image once. the highlighting is helpful, but i don't think it's helpful enough. not sure what others might think.

Reply:
also, if the picture were bigger (cropped to just the tasks), it would be easier to read the activity names and match them up to the descriptions you're giving.